packages/smart-forms-renderer/src/stories/itemTypes/Attachment.stories.tsx

```diff
@@ -31,9 +34,54 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+const targetlinkId = 'file-attachment';
+const targetText = 'File Attachment';
+
+const qAttachmentBasic = questionnaireFactory([
+  {
+    linkId: targetlinkId,
+    type: 'attachment',
+    repeats: false,
+    text: targetText
+  }
+]);
+
+const url = 'http://world_of_warcraft.com';
+const name = 'Vladimir';
+const fileName = 'foo.png';
+const type = 'image/png';
 
 export const AttachmentBasic: Story = {
   args: {
     questionnaire: qAttachmentBasic
+  },
+  play: async ({ canvasElement }) => {
+    await inputFile(
+      canvasElement,
+      targetlinkId,
+      [new File(['foo'], fileName, { type: type })],
+      url,
+      name,
+      'URL',
```

Comment:
add data-test attrs for `URL` and `File name (optional)` and hardcode tgem in inputFile instead